cmd/agent/core/agent.go

```diff
@@ -238,6 +238,8 @@ func run(ctx context.Context, c *cli.Command, backends []types.Backend) error {
 		return err
 	}
 
+	log.Debug().Any("labels", labels).Msgf("agent configured with labels")
```

Comment:
Maybe better to make it `info` and merge with line 285
> starting Woodpecker agent with version ... labels

Reply:
Normaly you as admin have set it so you should know this is for debugging porpuse